src/pages/publication/FilesAndLicensePanel.tsx

```diff
@@ -30,27 +37,33 @@ interface FilesAndLicensePanelProps {
 const FilesAndLicensePanel: React.FC<FilesAndLicensePanelProps> = ({ goToNextTab, uppy }) => {
   const { t } = useTranslation('publication');
   const { values }: FormikProps<Publication> = useFormikContext();
+  const [openLicenseModal, setOpenLicenseModal] = useState(false);
   const uploadedFiles = values[FilesFieldNames.FILES];
 
+  const toggleLicenseModal = () => {
+    setOpenLicenseModal(!openLicenseModal);
+  };
```

Comment:
Dette burde kanskje legges i en `InfoModal`-komponent? Så kan den passe på open/closed state selv og ta inn children som rendres inni modalen

Reply:
finnes en ButtonModal for just this purpose! 💃